lib/std/sha1.nim

```diff
@@ -8,6 +8,32 @@
 #
 
 ## Note: Import ``std/sha1`` to use this module
+##
+## SHA-1 (Secure Hash Algorithm 1) is a cryptographic hash function which
+## takes an input and produces a 160-bit (20-byte) hash value known as a
+## message digest.
+##
+## .. code-block::
+##    import std/sha1
+##
+##    let accessName = secureHash("John Doe")
+##    echo $accessName
+##    # --> F742544A66E7EF2310A42D0A01DBB5395F26BC0A
```

Comment:
In the long run these examples should become runnable so we should be using assert/doAssert here

Reply:
Changed to doAssert